lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -1533,6 +1533,15 @@ void DifferentiableActivityInfo::analyze(DominanceInfo *di,
               setVaried(cbi->getFalseBB()->getArgument(opIdx), i);
           }
         }
+        // Handle `switch_enum`.
```

Comment:
It seems necessary to also handle `switch_enum_addr`, but I didn't find Swift functions containing `switch_enum_addr` that also don't contain active enum values. A quick searches shows that `switch_enum_addr` is generated during later SIL passes and also for optional force-unwrapping.